lib/version_utils.pm

```diff
@@ -403,7 +403,7 @@ sub is_sle {
     return 1 unless $query;
 
     # Version check
-    return check_version($query, $version, qr/\d{2}(?:-sp\d)?/);
+    return check_version($query, $version, qr/\d{2}((?:-sp\d)?|(?:\.\d))/);
```

Comment:
```suggestion
    return check_version($query, $version, qr/1[25]-sp\d|16\.\d/);
```

Reply:
Unfortunately, this won't work with versions without service pack or minor version number. We still need to support `VERSION: XY` format

Reply:
But willng to consider possible also main version only, like `12` or `16`, the expr can be, instead:
`/1[25](-sp\d)?$|16(\.\d)?$/`